site/_posts/2020-09-22-manageiq-jansa-ga-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "ManageIQ Jansa GA announcement"
+author: Fryguy
+date: 2020-09-22
+tags: releases announcements
+comments: true
+published: true
+---
+TODO: [![ManageIQ Ivanchuk release](/assets/images/blog/Ivanchuk_GA_release.png)](/assets/images/blog/Ivanchuk_GA_release.png)
```

Comment:
This is TODO because this is like a banner that @cybette created previously, but I don't know how to create it yet.

Reply:
we only seem to have started having images with the Hammer release (because the name evoked an image).  I don't think we need an image for a release.

Reply:
we don't need a release banner, but just for reference, this tool makes it easy to create them: https://pablo.buffer.com/